components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -589,7 +590,16 @@ public static CloseableHttpResponse executeHTTPRequest(HttpRequestBase method, H
                 }
             }
         } while (retry);
-        return httpResponse;
+
+        if (httpResponse.getStatusLine().getStatusCode() == 200) {
+            return httpResponse;
+        } else {
+            httpResponse.close();
+            String errorMessage = EntityUtils.toString(httpResponse.getEntity(),
+                    APIConstants.DigestAuthConstants.CHARSET);
+            throw new ArtifactSynchronizerException(errorMessage + "Eevnthub status code is:"
```

Comment:
Typo and add a space

Reply:
https://github.com/wso2/carbon-apimgt/pull/9019/commits/29a2e4bba591e33476f9fcce64bd8668945123c4